platform/rct/hal/imu_stm32/imu_device_stm32.c

```diff
@@ -39,7 +39,7 @@
 #define IMU_TASK_PRIORITY	(tskIDLE_PRIORITY + 2)
 #define IS_9150_ADDR            0x68
 
-static struct is9150_all_sensor_data sensor_data[2];
+static struct is9150_all_sensor_data sensor_data[2] = {0};
```

Comment:
The gain is to make room for a larger LoggerConfig object in BSS (described in the top level comment).